UnitTests/SqlGeneration/IntegrationTests/CompoundMemberSqlBackendIntegrationTest.cs

```diff
@@ -139,7 +139,7 @@ where someIDs.Contains (c.KnifeID)
               .With.Message.EqualTo (
                   "The SQL 'IN' operator (originally probably a call to a 'Contains' method) requires a single value, so the following expression cannot be "
                   + "translated to SQL: 'new MetaID(Value = [t0].[KnifeID] AS Value, ClassID = [t0].[KnifeClassID] AS ClassID) "
-                  + "IN (value(Remotion.Linq.SqlBackend.UnitTests.TestDomain.MetaID),value(Remotion.Linq.SqlBackend.UnitTests.TestDomain.MetaID))'."));
+                  + "IN (Remotion.Linq.SqlBackend.UnitTests.TestDomain.MetaID,Remotion.Linq.SqlBackend.UnitTests.TestDomain.MetaID)'."));
```

Comment:
Not sure what's gonna happen here sicne we don't do literals. perhaps they become parameters?

Reply:
This is not SQL syntax, but `Expression.ToString()` - the array containing the two `MetaID` objects is now a `ConstantCollectionExpression`, which uses a different string representation than the previous `SqlCollectionExpression`.
The `$"value({item.ToString()})"` indicated that each item would be treated as a separate value (resulting in a parameter for each item), whereas now we treat the whole collection as a single value, resulting in a single parameter.

Reply:
Yeah. Should have been more precise: I didn't think deeply on it, simply noted that this will likely change once we get to the updated code.